casbin/ip_parser/parser/byte.h

```diff
@@ -1,7 +1,8 @@
 #ifndef IP_PARSER_PARSER_BYTE
 #define IP_PARSER_PARSER_BYTE
 
-typedef unsigned short int byte;
+using namespace std;
+typedef unsigned short int Byte;
```

Comment:
short int is 16 bits, why name it `Byte`, didn't notice this before

Reply:
It is described by the idea of @​​divy9881 and conflicts with c++17 std::byte